addons/spreadsheet/static/src/chart/odoo_chart/odoo_line_chart.js

```diff
@@ -126,5 +127,21 @@ function getLineConfiguration(chart, labels, locale) {
     if (chart.stacked) {
         config.options.scales.y.stacked = true;
     }
+
+    const formatCallback = (value) => {
```

Comment:
once https://github.com/odoo/o-spreadsheet/pull/4728/files is merged, we could export the new helper from the lib and use it here